### ISSUE TITLE
core: Load-based lease/replica rebalancing

### ISSUE THREAD

**Issue by jseldess** [cockroachdb/docs#1555]
Background: 
 - https://airtable.com/tblD3oZPLJgGhCmch/viwdV1sM5KEK2L14K/recX1W3voJTcTCrzp
 - RFC: https://github.com/cockroachdb/cockroach/blob/master/docs/RFCS/rebalancing_for_1_1.md

PRs: None yet

**Comment by jseldess** [comment]
@a-robinson , once this is ready for docs, please assign the `ready` label and add relevant PR links.

**Comment by jseldess** [comment]
Further explanations/guidance: [explanations.pdf](https://github.com/cockroachdb/docs/files/1234728/explanations.pdf)

**Comment by a-robinson** [comment]
For what it's worth, that doesn't explain load-based replica rebalancing, just lease rebalancing.

Also, thanks for the reminder -- this is ready to document. The planning was done in https://github.com/cockroachdb/cockroach/pull/16296, with most of the implementation in https://github.com/cockroachdb/cockroach/pull/16878 and an upgrade related detail in https://github.com/cockroachdb/cockroach/pull/17695

**Comment by jseldess** [comment]
Ah, right. Thanks for the correction, @a-robinson.

**Comment by a-robinson** [comment]
Update: this rebalancing replicas based on size and load has been disabled by default in 1.1, so documenting it should be considered lower priority and/or pushed back to 1.2.

**Comment by jseldess** [comment]
Closing in favor of https://github.com/cockroachdb/docs/issues/2051.
